### PR TITLE
Add create account method

### PR DESCRIPTION
**Pull request by xiaoge26**
Fixed Checkstyle errors

### CODE REVIEW

**Comment by xiaoge26** [comment]
I will fix the errors soon.

**Comment by xiaoge26** [comment]
Checkstyle issues fixed!

**Comment by xiaoge26** [comment]
Checkstyle issues Fixed!

**Review by tyuyang — APPROVED** [review]
All checks passed. Merging now.

File: src/main/java/seedu/bankwithus/AccountList.java [hunk 1]
```diff
@@ -1,14 +1,32 @@
 package seedu.bankwithus;
 
+import java.util.ArrayList;
 import java.util.Scanner;
 
 public class AccountList {
-
+    protected ArrayList<Account> accounts;
```
Comment (tyuyang) [inline src/main/java/seedu/bankwithus/AccountList.java:7]: Why... you might as well name it public since everything is in the same package...

Comment (xiaoge26) [reply src/main/java/seedu/bankwithus/AccountList.java:7]: Ok I'll change it later!

Comment (tyuyang) [reply src/main/java/seedu/bankwithus/AccountList.java:7]: Lol wait no. I was writing these reviews as a joke. And plus I intention was for this to be changed from protected to private, not public

Comment (xiaoge26) [reply src/main/java/seedu/bankwithus/AccountList.java:7]: Ok sure! We may need to standardise these in the next version.

**Pull request merged**